server/src/main/java/io/deephaven/server/session/SessionState.java

```diff
@@ -761,6 +761,7 @@ private synchronized void setState(final ExportNotification.State state) {
                 if (errorId == null) {
                     assignErrorId();
                 }
+                //TODO do we want logs about this?
                 safelyExecute(() -> errorHandler.onError(state, errorId, caughtException, dependentHandle));
             }
```

Comment:
TODO added because this is the only remaining callsite for safelyExecute - like the python autocomplete code, I'm thinking we'll replace it with a try/catch. But do we want to log the error in this case, or are we only intending to propagate it to the client?

Reply:
IMO, SessionState changes belong in a stream table to make client debugging a lot easier. I actually have a branch that does this. It has been helpful to debug api client issues for the c++ and go clients. I have not yet put it up for a PR as the web-ui is extremely annoying (re-creating exports of the lastBy version of the table makes the table unusable without also filtering out the web-ui's session). State change logging is extremely verbose and hard to follow from the log -- so the table is a real winner in that regard. If we only log errors here (and not requests and other state changes) then there is not enough context here to understand what failed or why. Most errors will have already logged if they were not due to API issues (i.e. uncaught exceptions will be logged, but illegal parameters and other StatusRuntimeException will not be logged). 

I prefer safely ignoring any onError failure here.